src/server/helpers/config.js

```diff
@@ -11,6 +11,8 @@ import dotenv from 'dotenv';
 const CLIENT_SIDE_WHITELIST = [
   'NODE_ENV',
   'BASE_URL',
+  'POLLER_REQUEST_BUILDS',
```

Comment:
Any particular reason for exposing it to client UI?
To make sure we could display it in the UI?

Reply:
I'm not sure if I understand.

`CLIENT_SIDE_WHITELIST` is a list of config variables that are whitelisted to be sent to client app in the browser. It's meant to make part of the config shared between browser and server.

This means that all the variables in `CLIENT_SIDE_WHITELIST` are sent in HTML to the browser and are available by JavaScript there:
![screen shot 2017-08-28 at 10 56 48](https://user-images.githubusercontent.com/83575/29766692-e3ec3c80-8bdf-11e7-8650-a19042fc4084.png)

At this stage it doesn't seem that we need to expose `POLLER_REQUEST_BUILDS` or `POLLER_BUILD_THRESHOLD` to client side app.

I also don't think that adding them to this list is needed to make it possible to override them in command line.
I tried running `npm run poll-repositories -- --env=environments/dev.env --POLLER_BUILD_THRESHOLD=1` and it correctly overridden value of this var even if it was not in `CLIENT_SIDE_WHITELIST`.

If by any chance `CLIENT_SIDE_WHITELIST` is needed for anything else then exposing config items to client side app it's a bug somewhere in our config code.

Reply:
@bartaz You are right, I misunderstood the discrete list of arguments mechanics. `yargs` does the heavy-lifting to process.env on its own. Thanks for pointing this out.